src/activy/register.py

```
@@ -167,12 +175,156 @@ def registerStage9(device, templates):
+            raise TimeoutError(f"Confirm popup didnt appear.")
+        # Click Next (confirm joining the workplace)
+        clickNodeByClassInstance(device, "android.view.View", 6)
+    else:
+        raise ValueError(f"Challenge '{workplace_name}' not found")
+
+    # Check stage, wait for the join workplace prompt to disappear
+    wait_for_element(device, contentDesc=workplace_name)
+
+    for _ in range(60):
+        img = device.screenshot()
+        pixel = img.getpixel((5, img.height // 2)) # Get pixel at 5px from the left and in the middle of the screen
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)